LSE-61.tex

```diff
@@ -1174,6 +1178,93 @@ \subsubsection{All-Sky Visualization of Data Releases}
+
+\textbf{Specification:}
+The Data Management system shall include a secure and authenticated Internet endpoint for an IVOA-compliant HiPS service.  This service shall be advertised via Registry as well as in the HiPS community mechanism operated by CDS, or whatever equivalent mechanism may exist in the LSST operations era.
+
+\textbf{Discussion:}
+The DM HiPS service will be available only to data rights holders.  LSST EPO will also operate a world-public HiPS service, but with its spatial resolution limited to approximately one arcsecond.
+
+\paragraph{HiPS Linkage to Coadds}\hfill  % Force subsequent text onto new line
+
+\label{DMS-REQ-0381}
+\textbf{ID:} DMS-REQ-0381
+
+\textbf{Specification:}
+The HiPS maps produced by the Data Management system shall provide for straightforward linkage from the HiPS data to the underlying LSST coadded images.  This SHOULD be implemented using a mechanism supported by both the LSST Science Platform and by community tools.
+
+\textbf{Discussion:}
+It is intended that this be done using the "HiPS Progenitor" mechanism introduced at the May 2018 IVOA meeting, or an evolution of it that emerges from the IVOA standardization process.
+
+\paragraph{HiPS Visualization}\hfill  % Force subsequent text onto new line
+
+\label{DMS-REQ-0382}
+\textbf{ID:} DMS-REQ-0382
+
+\textbf{Specification:}
+The LSST Science Platform shall support the visualization of the LSST-generated HiPS image maps as well as other HiPS maps which satisfy the IVOA HiPS Recommendation, and shall provide integrated behavior, such as the overplotting of catalog entries, comparable to that provided for individual source images (e.g., PVIs and coadd tiles).
+
+\textbf{Discussion:}
+Further details will be provided in the LSST Science Platform Requirements, \citeds{LDM-554}.
+
+\paragraph{Produce MOC Maps}\hfill  % Force subsequent text onto new line
+
+\label{DMS-REQ-0383}
+\textbf{ID:} DMS-REQ-0383
+
+\textbf{Specification:}
+Data Release Production shall include the production of Multi-Order Coverage maps for the survey data, conformant with the IVOA MOC recommendation.  A separate MOC shall be produced for each filter band for the main survey.  Additional MOCs SHOULD be produced to represent special-programs datasets and other collections of on-sky data.
+
+\textbf{Discussion:}
+It is likely to be useful to produce quite a large number of MOCs as part of releasing the data and documenting its quality.  For example, it may be useful to produce both a MOC for <em>all</em> the data from a band and for only that part of the sky for which the SRD requirements in that band have been met.  It also seems useful to produce MOCs for the deep drilling fields, etc.  It may also be useful to produce MOCs on, for instance, a nightly basis, reflecting that part of the sky for which coverage was obtained in that night.  The LSST project should engage in the work begun in 2018 on the development of standards and tools for <em>spatiotemporal</em> MOCs.
```

Comment:
There are stray `<em>` tags in the text here.

Reply:
Yes. I need to work out why the post-processing Python script is not spotting those.